modules/integration/tests-common/admin-clients/src/main/java/org/wso2/mb/integration/common/clients/operations/clients/AndesAdminClient.java

```diff
@@ -162,4 +162,30 @@ private void configureCookie(ServiceClient client) throws AxisFault {
                                sessionCookie);
         }
     }
+
+    /**
+     * Get dead letter channel queue
+     *
+     * @return queue
+     * @throws AndesAdminServiceBrokerManagerAdminException
+     * @throws java.rmi.RemoteException
+     */
+    public Queue getDlcQueue() throws AndesAdminServiceBrokerManagerAdminException,
+                                      java.rmi.RemoteException {
+
+        Queue[] queueList = stub.getAllQueues();
+        Queue dlcQueue = null;
+
+        if (queueList != null) {
+            for (Queue queue : queueList) {
+                String nameOfQueue = queue.getQueueName();
+                if (nameOfQueue != null && nameOfQueue.contains("DeadLetterChannel")) {
```

Comment:
You can use a constant for "DeadLetterChannel". Check whether andes has a constant for this which we can access here.